test/auto-fix-func.js

```diff
@@ -53,12 +39,15 @@ testRule(
       },
     ],
 
-    accept: [
-      { code: '.foo { color: #fff; }' },
-      { code: '.foo { color: red; }' },
-    ],
-
     reject: [
+      {
+        code: '.foo { color: #fff; }',
+        fixed: '.foo { color: $color-white; }',
+      },
+      {
+        code: '.foo { color: red; }',
+        fixed: '.foo { color: $color-red; }',
+      },
       {
         code: '.foo { font-size: 16px; }',
         message: `"font-size" is not a color property (${ruleName})`,
```

Comment:
- [ ] Custom autofix error message can't be asserted yet.
   https://github.com/stylelint/jest-preset-stylelint/issues/60